pandas/core/generic.py

```diff
@@ -7874,10 +7874,8 @@ def groupby(
             If False: show all values for categorical groupers.
 
             .. versionadded:: 0.23.0
-
-        **kwargs
-            Optional, only accepts keyword argument 'mutated' and is passed
-            to groupby.
+        mutated : bool, False
+            `mutated` is passed to groupby.
```

Comment:
I'm not sure what ``mutated`` actually does. It's tested for in ``window/test_grouper.py`` so assume it has something to do with windowing ops. anyone got a suggestion for a better doc string?